src/compute/src/sink/persist_sink.rs

```diff
@@ -221,19 +208,19 @@ where
         &persist_oks,
         &persist_errs,
         Arc::clone(&persist_clients),
+        compute_state.read_only_rx.clone(),
     );
 
-    let (append_frontier_stream, append_token) = append_batches(
+    let (_append_frontier_stream, append_token) = append_batches(
```

Comment:
Can we remove this output if it's not needed?

Reply:
👌